packages/@aws-cdk/aws-dynamodb/lib/table.ts

```diff
@@ -247,6 +254,7 @@ export class Table extends Resource {
       streamSpecification: props.stream ? { streamViewType: props.stream } : undefined,
       timeToLiveSpecification: props.timeToLiveAttribute ? { attributeName: props.timeToLiveAttribute, enabled: true } : undefined
     });
+    this.table.applyRemovalPolicy(props.removalPolicy, { default: RemovalPolicy.RETAIN });
```

Comment:
Don’t specify the default. It is “by default” retain but we may introduce an option to change the default within a scope/stack (eg for dev stacks). We should improve the docs and/or an enumeration value that says “inherit”.

Reply:
Okey.